internal/common/objects/tests/megabikes_test.go

```diff
@@ -1,206 +1,206 @@
-package objects

```

Comment:
I don't know why this file got rewritten :sob: it's probably full of CRLFs or something